include/onnxruntime/core/graph/graph_viewer.h

```diff
@@ -146,6 +150,17 @@ class GraphViewer {
   */
   bool IsConstantInitializer(const std::string& name, bool check_outer_scope) const;
 
+  /** returns the initializer's TensorProto if 'name' is an initializer, is constant and
+  cannot be overridden at runtime. If the initializer is not found or is not constant, a nullptr is returned.
+  @param check_outer_scope If true and the graph is a subgraph,
+         check ancestor graph/s for 'name' if not found in 'graph'.
+  @remarks check_outer_scope of true is not supported in a minimal build
```

Comment:
>check_outer_scope of true is not supported in a minimal build [](http://example.com/codeflow?start=11&length=61)

is this still true? same for Graph::GetConstantInitializer(). I was trying to find out why it is unsupported but didn't manage to

Reply:
This is removed in this PR #5077 and seems added back in #5433, for `IsOuterScopeValue`, we probably don't need this remarks anymore